src/main/kotlin/no/nav/familie/ba/sak/behandling/vedtak/VedtakService.kt

```diff
@@ -187,6 +196,27 @@ class VedtakService(private val behandlingService: BehandlingService,
         return vedtak.vedtakBegrunnelser.toList()
     }
 
+    private fun erSammeMåned(date0: LocalDate?, date1: LocalDate?): Boolean =
+            date0 != null && date1 != null && date0.toYearMonth() == date1.toYearMonth() || date0 == null && date1 == null
+
+    private fun hentPersonerManglerOpplysninger(vilkårsvurdering: Vilkårsvurdering, fom: LocalDate, tom: LocalDate?, vedtakBegrunnelseType: VedtakBegrunnelseType, visOpphørsperioderToggle: Boolean): List<Person> =
```

Comment:
```suggestion
    private fun hentPersonerManglerOpplysninger(vilkårsvurdering: Vilkårsvurdering, vedtaksperiodeFom: LocalDate, vedtaksperiodeTom: LocalDate?, vedtakBegrunnelseType: VedtakBegrunnelseType, visOpphørsperioderToggle: Boolean): List<Person> =
```
maybe add some description to easier keep track of all the dates?

Reply:
You mean add some comments?

Reply:
just giving it a more describing name, as suggested 😄